packages/stylelint-config/scss.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	extends: ['./index.js'],
+	rules: {
+		'scss/at-rule-no-unknown': [true, { ignoreAtRules: ['mixin', 'define-mixin'] }],
```

Comment:
Aren't you missing removing this rule from `index.js`?

Reply:
@nicholasio what a dumb mistake on my end! You're right! I took it a step forward with the exports, let me know your thoughts!